docs/Installation/System-Requirements.md

```diff
@@ -0,0 +1,31 @@
+title: System Requirements
+description: System requirements to sync and run Pantheon 
+<!--- END of page meta data -->
+
+# System Requirements 
+
+The system requirements for Pantheon depend on a number of factors: 
+
+* Network - are you connecting to a public Ethereum network (eg, mainnet) or a private network 
+* Network size if connecting to a private network 
+* Number and complexity of queries being performed by node 
+
+## CPU Requirements for Syncing to Ethereum Mainnet
+
+Syncing to the Ethereum mainnet using a t3.xlarge takes about 15 hours.  The sync to mainnet can be completed with 
+a t3.large but takes longer.  
```

Comment:
I wouldn't include an expected fast sync time - it's hugely variable depending on which nodes you find and if the pivot block has to be updated.

The required instance type is far more complex than this. An `i3.xlarge` can sync MainNet reliably but has very little extra capacity to service JSON-RPC queries. An `i3.2xlarge` is faster to sync and has more headroom.  A `t3.2xlarge` can also sync main net but is slower to sync than either of the `i3` instances.  We haven't got any meaningful info on other `t3` category instance sizes.

Reply:
Removed with the reworking